api/v1alpha1/annotations.go

```diff
@@ -4,9 +4,15 @@ const (
 	// AnnotationKeyPrefix is the prefix used for annotation key values
 	AnnotationKeyPrefix = LabelKeyPrefix
 
-	// SSOUserIDAnnotationKey is used to store the user's user_id claim value issued by the SSO provider
-	SSOUserIDAnnotationKey = AnnotationKeyPrefix + "sso-user-id"
+	// UserIDUserAnnotationKey is used to set an annotation value in the User resource on the member cluster, that
+	// contains the user's User ID as set in the user's JWT token.
+	UserIDUserAnnotationKey = AnnotationKeyPrefix + "sso-user-id"
 
-	// SSOAccountIDAnnotationKey is used to store the user's account_id claim value issued by the SSO provider
-	SSOAccountIDAnnotationKey = AnnotationKeyPrefix + "sso-account-id"
+	// AccountIDUserAnnotationKey is used to set an annotation value in the User resource on the member cluster, that
+	// contains the user's Account ID as set in the user's JWT token.
+	AccountIDUserAnnotationKey = AnnotationKeyPrefix + "sso-account_id"
```

Comment:
I guess you didn't intend to change the actual annotation. It would break our pendo UI.
```suggestion
	AccountIDUserAnnotationKey = AnnotationKeyPrefix + "sso-account-id"
```

Reply:
You're right, thanks for spotting this.